android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -452,6 +452,13 @@ public void onActivityResult(Activity activity, int requestCode, int resultCode,
     else
     {
       imageConfig = getResizedImage(reactContext, this.options, imageConfig, initialWidth, initialHeight, requestCode);
+      
+      // some devices allow video anyways in photo only mode e.g Huawei Mate 20 Lite
+      if (imageConfig == null) {
+        responseHelper.invokeError(callback, "Video is not supported in image mode");
```

Comment:
Is a user selecting a video the only reason it can be null?

Reply:
I don't know :/  - seems so as it's not an image

Reply:
If you can investigate this more and see if there are any other reasons could be null and then update the error message to something more generic (if needed), then I can get this merged.

Relavant code: 
https://github.com/react-native-community/react-native-image-picker/blob/53316af68445de02a274fb53a7f2cdd0a59a431a/android/src/main/java/com/imagepicker/utils/MediaUtils.java#L81-L200

Reply:
@Johan-dutoit this is still an issue in 2020. I guess keeping this opened for one year because of not enough generic error message is also not a correct way of resolving this, because of tons of affected production apps. Any chance to get this merged?

Reply:
Just tested this code. It was still crashing for me.

I changed line 458 to `responseHelper.putString("error", "Video is not supported in image mode");`

and before calling `imageConfig.saveToCameraRoll` I checked for `imageConfig != null`

All is working well with this.

Reply:
@MorganTrudeau is this what you mean? Does it work for you? I haven't touched React native in a couple months nor do have the phone to reproduce this anymore :/